### PR TITLE
add redis config, move elasticsearch to initializers

### PR DESCRIPTION
**Pull request by pkarman**

### CODE REVIEW

File: config/initializers/redis.rb [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+# we assume defaults are fine for test/development.
+# https://github.com/mperham/sidekiq/wiki/Using-Redis
+if ENV["RACK_ENV"] == "production"
+  vcap = ENV["VCAP_SERVICES"]
+  vcap_config = JSON.parse(vcap)
+  vcap_config.keys.each do |vcap_key|
+    if vcap_key.match(/redis\d+-swarm/)
+      redis_config = vcap_config[vcap_key][0]['credentials']
+      redis_url = "redis://#{redis_config['password']}@#{redis_config['hostname']}:#{redis_config['port']}"
+      #STDERR.puts "REDIS_URL=#{redis_url}"
```
Comment (jessieay) [inline config/initializers/redis.rb:11]: was this left in accidentally?

File: Procfile [hunk 2]
```diff
@@ -1,2 +1,2 @@
-worker: bundle exec sidekiq -C config/sidekiq.yml -r ./lib/app.rb
+worker: bundle exec sidekiq -C config/sidekiq.yml
```
Comment (jessieay) [inline Procfile:1]: should we move the sidekiq config while we're here?

File: config/initializers/redis.rb (hunk 1 above)
Comment (pkarman) [inline config/initializers/redis.rb:11]: no, but I understand why you note it. I intended to use it for testing #40 when I got to that point. I can zap it for now if you like.

File: Procfile (hunk 2 above)
Comment (pkarman) [inline Procfile:1]: to where? it's analogous to `config/database.yml`.

Comment (jessieay) [inline Procfile:1]: make sense, let's keep as-is

File: config/initializers/redis.rb (hunk 1 above)
Comment (jessieay) [inline config/initializers/redis.rb:11]: nah let's just keep it

**Comment by jessieay** [comment]
thanks for getting back to me on the comments! merging

**Pull request merged**